zokrates_core/src/proof_system/bn128/g16.rs

```diff
@@ -20,25 +29,42 @@ impl G16 {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+struct G16ProofPoints {
+    a: G1PairingPoint,
+    b: G2PairingPoint,
+    c: G1PairingPoint,
+}
+
+impl G16ProofPoints {
+    fn new(a: G1PairingPoint, b: G2PairingPoint, c: G1PairingPoint) -> Self {
+        G16ProofPoints { a, b, c }
+    }
+}
+
+impl Proof<G16ProofPoints> {
+    fn from_json(str: &str) -> Self {
```

Comment:
same here

Reply:
Moved this method to https://github.com/Zokrates/ZoKrates/pull/583/files#diff-189d3ba6c93acd77029cea5c8a6a386aR26